pages/index.js

```diff
@@ -1,31 +1,22 @@
 import React from 'react';
 import Link from 'next/link';
-import styled, { css } from 'react-emotion';
+import { bodyStyle } from './styles/GlobalStyles';
```

Comment:
Make it a styled component instead

Reply:
Yes it's better to have the style with the component. It's one of the advantage of css in js